### PR TITLE
`bugs-tab`: Use information from page

### PR DESCRIPTION
**Pull request by fregante**
Fixes https://github.com/sindresorhus/refined-github/issues/2889#issuecomment-599179639

This code is a mess though, every time I try to get data from the DOM _and_ the API it's more complex than I thought.

I'm not sure I'll merge this, the cache is set to expire after 30 minutes already, it's not the end of the world.

### CODE REVIEW

File: source/features/bugs-tab.tsx [hunk 1]
```diff
@@ -18,11 +23,13 @@ const countBugs = cache.function(async (): Promise<number> => {
...
-}, {
-	maxAge: 1 / 24 / 2, // Stale after half an hour
-	staleWhileRevalidate: 4,
-	cacheKey: (): string => __featureName__ + ':' + getRepoURL()
-});
+}, cacheOptions);
+
+function cleanForComparison(searchQuery: SearchQuery): SearchQuery {
+	const newQuery = searchQuery.clone();
+	newQuery.remove('sort:updated-desc', 'is:open', 'is:closed');
```
Comment (busches) [inline source/features/bugs-tab.tsx]: Do we need to remove @me and author:

**Comment by fregante** [comment]
No, we need to match the exact query because anything else will change the number. 

`newsfeed label:bug` will show different results and result counts, that’s why this comparison is so complex

File: source/features/bugs-tab.tsx (hunk 1 above)
Comment (fregante) [reply source/features/bugs-tab.tsx]: Only these queries don’t change the numbers on the page. `author` does

**Comment by fregante** [comment]
I think I'll just make an API to force an refresh from the source, this method isn't sustainable.

Example:

```js
const isBugsPage = new SearchQuery(location).includes('label:bug');
const countPromise = isBugsPage ? countBugs.skipCache() : countBugs();
```

**Pull request closed without merging**